### ISSUE TITLE
Use unicode characters directly?

### ISSUE THREAD

**Issue by ronyeh** [0xfe/vexflow#1087]
https://github.com/0xfe/vexflow/blob/2315e7aba40a9bd0328ccb0b6c155eeff4489417/src/tables.ts#L735-L747

Minor thing I noticed. Can we replace the unicode table with the actual unicode characters, since source files support utf-8?

The following are all equivalent:
```
String.fromCharCode(parseInt('00F8', 16))
String.fromCharCode(0x00F8)
'ø'
```

Ideally we use the bottom one. We can keep the hex number code point in a comment above each entry.

**Comment by ronyeh** [comment]
So the new unicode table would look like

```
unicode: { 
  // 0x266F
  sharp: "♯",
  // 0x266D 
  flat: "♭", 
  // 0x266E
  natural: "♮", 
```

**Comment by mscuthbert** [comment]
I think this is a good idea until you get to Astral plane ( > 0xFFFF) characters which some Javascript engines have a hard time with.  So sharp, flat, natural are easy, but double-flat and double-sharp are astral characters which need to be constructed more gently.

**Comment by 0xfe** [comment]
I would prefer it the other way round -- put the actual characters in the comments. Unicode breaks in too many ways particularly on older tooling or misconfigured apps.

**Comment by ronyeh** [comment]
Done. I put the unicode characters in comments. But I removed the `parseInt()` in favor of using hex numbers directly. Hopefully JS engines don't have difficulty with 0xFFF style numbers.

Also, my text labels are inspired by the SMuFL naming convention. I understand the triangle is originally just a major triad, but SMuFL calls it major seventh so I went with that.

https://github.com/0xfe/vexflow/blob/a4a528d8991fbaf6899decd3208b4453ebbb5d5c/src/tables.ts#L731-L746
